lisa/tests/kernel/test_bundle.py

```diff
@@ -155,14 +154,61 @@ class CannotCreateError(RuntimeError):
     """
     pass
 
-class TestBundle(Serializable, abc.ABC):
+class TestBundleBase(Serializable):
```

Comment:
Why create this split? I only had a quick look but I didn't find a user for this (there's LoadTrackingBase but it implements _from_testenv for some reason).

I also strongly believe all of our TestBundles should have the `from_testenv` method. It's an alternative constructor after all, if for some reason some ugly duckling doesn't want it (and I don't see a reason why we'd even have one like this), we can document/raise somewhere in there.